lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -26,8 +26,8 @@
        or by using the use_proxy option.
      - HTTP redirects can redirect from HTTP to HTTPS so you should be sure that
        your proxy environment for both protocols is correct.
-     - From Ansible 2.4 when run with C(--check), it will do a HEAD request to validate the URL but
-       will not download the entire file or verify it against hashes.
+     - From Ansible 2.4 when run with C(--check), it will do a GET request to validate the URL and
```

Comment:
That's obviously wrong, since older versions like 2.4 won't change.

Reply:
You're quite right, my apologies.  I'm pushing a new version which adds a new comment, and I'll add a changelog fragment too.